packages/core/graphql/package.json

```diff
@@ -9,26 +9,26 @@
   "scripts": {
     "dev": "yarn gen && parcel watch",
     "build": "yarn gen && parcel build",
-    "gen": "graphql-codegen --config codegen.yml"
+    "gen": "graphql-codegen --config codegen.yml && ed -s src/generated/index.ts <<< $'2d\nw'"
```

Comment:
This is a workaround for https://github.com/dotansimha/graphql-code-generator/issues/9440